methods/count/tests.js

```diff
@@ -0,0 +1,17 @@
+import test from 'ava';
+import count from '.';
+
+test('returns the number of elements', t => {
+    const array = ['a', 'b', 'c'];
+    t.is(count(array), 3);
+});
+
+test('returns the number of elements that equal the argument', t => {
+    const array = ['a', 'b', 'c'];
+    t.is(count(array, e => e === 'b'), 1);
+});
+
+test('returns the number of element for which the block evaluates to true', t => {
+    const array = ['a', 'b', 'c'];
+    t.is(count(array, e => e !== 'b'), 2);
+});
```

Comment:
@aldnav This is lacking this feature of Ruby's `count` method: `count(obj) → int`

> If an argument is given, counts the number of elements which equal `obj` using `==`.

Reply:
@arnellebalane nice catch. Thanks. Added here a79b630